### DISCUSSION TITLE
dist folder not exists

### DISCUSSION THREAD

**Discussion by Deorigami** [varabyte/kobweb#535] [Q&A]
Hi there,
I try to add kobweb for my Multiplatform project for my front end web.
After following the step on Readme
i got this error when running kobweb run

> Caused by: org.gradle.api.GradleException: e: Your .kobweb/conf.yaml dev script ("build/dist/js/developmentExecutable/front_end.js") could not be found. This will cause the page to fail to load with a 500 error. Perhaps search your build/ directory for "front_end.js" to find the right path.

if i run fresh project the dist folder is exists, but if i run from my multiplatform project, it doesnt exists

**Comment by bitspittle** [comment]
Without knowing the structure of your site, I'm not sure what's going on. You should have a `site` module with the `.kobweb/conf.yaml` file in there; its `build.gradle.kts` file should be set up already; and the `build` folder should live under the `site` folder.

Maybe check out this section to see if it shares any advice that applies to your situation?
https://github.com/varabyte/kobweb#adding-kobweb-to-an-existing-project

**Comment by Deorigami** [comment]
![Screenshot 2024-05-18 at 05 20 27](https://github.com/varabyte/kobweb/assets/70903634/f0998648-a1c0-4ffc-97e6-e9c61e6bc548)

Actually after some digging, the folder dist is not created when embedding kobweb into KMP project, and i change dist folder to kotlin-webpack and it runs ..

**Reply by bitspittle** [reply]
Interesting. Thanks for sharing the update.

What version of Compose are you using? And are you using K2?

**Reply by Deorigami** [reply]
Im using kotlin 2.0.0rc3 and compose multiplatform 1.6.10rc3

**Reply by Deorigami** [reply]
My goals is to reuse the business logic inside kobweb .. and it quite success, i could use and inject my existing logic into kobweb

**Reply by bitspittle** [reply]
Ah yes you're ahead of what we recommend but I understand your situation.

I was vaguely aware that in the next version of Compose they moved build stuff around and it's annoying because we'll have to migrate users over to the new build locations. But I will look into it more when 1.6.10 is released.

Anyway, just fixing it by hand was correct, since you're working a little bit ahead of us.
